<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Inmanage</class>
 <widget class="QWidget" name="Main_Inmanage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main_Inmanage</string>
  </property>
  <widget class="QLabel" name="video_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>45</y>
     <width>670</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>2</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>停车场视频实时监控画面</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>5</y>
     <width>400</width>
     <height>530</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="carInfo_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>当前车场车辆实时信息</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="carTotal_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>目前车场总车辆：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="freePark_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>目前车场空闲车位：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="snap_btn">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>车辆特征图片抓拍</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="record_btn">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>停车场监控手动录制</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QRadioButton" name="auto_btn">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>视频监控每30秒自动录制</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
